### PR TITLE
[TOOLS-4352] NullPointException occurred during comment migration

### PR DESCRIPTION
**Pull request by Kang-dot**
http://jira.cubrid.org/browse/TOOLS-4352

- Purpose
fix NullPointException and add getColumnConfigIgnoreCase method

- Implementation
N/A

- Remarks
N/A

### CODE REVIEW

**Review by kisoo-han — APPROVED** [review]

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java [hunk 1]
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
...
+				}
+			} else {
+				tarTbl.setComment("");
 			}
 			for (Column tarCol : tarTbl.getColumns()) {
-				SourceColumnConfig sourceCol = setc.getColumnConfig(tarCol.getName().toUpperCase());
-				if (tarCol.getComment() == null || sourceCol.getComment() != tarCol.getComment()){
-					tarCol.setComment(sourceCol.getComment());
+				SourceColumnConfig sourceCol = setc.getColumnConfig(tarCol.getName());
+				if (sourceCol.getComment() != null){
```
Comment (hwany7seo) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java]: sourceCol  can be null.

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java [hunk 2]
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
 				continue;
 			}
 			Table tarTbl = cfg.getTargetTableSchema(setc.getTarget());
-			if (tarTbl.getComment() == null || tarTbl.getComment() != setc.getComment()) {
-				tarTbl.setComment(setc.getComment());
+			if (setc.getComment() != null) {
```
Comment (hwany7seo) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java]: also setc can be null.

**Review by hwany7seo — COMMENTED** [review]
please check comment.

File: com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java [hunk 3]
```diff
@@ -258,13 +258,21 @@ protected void setDDLText() {
 				continue;
 			}
 			Table tarTbl = cfg.getTargetTableSchema(setc.getTarget());
```
Comment (airnet73) [inline com.cubrid.cubridmigration.ui/src/com/cubrid/cubridmigration/ui/wizard/page/BaseConfirmationPage.java:260]: If tarTbl is null, it seems necessary to check.

**Review by airnet73 — APPROVED** [review]

**Review by hwany7seo — APPROVED** [review]

**Pull request merged**